llvm/lib/Target/SPIRV/SPIRVEmitIntrinsics.cpp

```diff
@@ -2367,6 +2371,29 @@ void SPIRVEmitIntrinsics::applyDemangledPtrArgTypes(IRBuilder<> &B) {
   }
 }
 
+GetElementPtrInst *
+SPIRVEmitIntrinsics::simplifyZeroLengthArrayGepInst(GetElementPtrInst *GEP) {
```

Comment:
I think this is OK for us:
 - `RWStructuredBuffer` uses `[0 x T]`, but it never gets converted to a GEP
 - unbounded arrays are not allowed inside those buffers so no GEP with `[0 x T]`
 - `RWStructuredBuffer arrayOfBuffer[]` is not implemented yet, but would probably not use an llvm runtime array?
 
@s-perron am I missing something?